jdaviz/configs/default/plugins/subset_tools/tests/test_subset_tools.py

```diff
@@ -1,22 +1,23 @@
 import warnings
-import pytest
 
 import numpy as np
+import pytest
+from astropy import units as u
 from astropy.nddata import NDData
-import astropy.units as u
-from regions import CirclePixelRegion, PixCoord
-from specutils import SpectralRegion
+from glue.core.edit_subset_mode import ReplaceMode, OrMode, NewMode
 from glue.core.roi import EllipticalROI, CircularROI, CircularAnnulusROI, RectangularROI
-from glue.core.edit_subset_mode import ReplaceMode, OrMode
 from numpy.testing import assert_allclose
+from regions import CirclePixelRegion, PixCoord
+from specutils import SpectralRegion
 
 from jdaviz.configs.default.plugins.subset_tools import utils
 from jdaviz.core.region_translators import regions2roi
 
 
-@pytest.mark.filterwarnings('ignore')
```

Comment:
Should avoid global warning ignore if we can help it, which we actually can, so I fixed it up here even though it is unrelated to JDAT-5014